rov/movement/thruster/Force_Limiter.py

```diff
@@ -0,0 +1,117 @@
+from math import floor
+import operator
+
+class ForceLimiter(object):
+
+    # TODO: convertToPowerUnit()
+
+    def __init__(self):
+        self.a6vector = [0 for _ in range(6)]
+        self.an8vector = [0 for _ in range(8)]
+        self.last8vector = [0 for _ in range(8)]
+
+        self.mapper = Mapper()
+
+    """
+    1. acceptable_total_power
+    2. acceptable_individual_total
+    3. acceptable_total_change
+    4. acceptable_individual_change
+    """
+
+    def convertToPowerUnit(self, an8vector):
+        # equation: PWM vs Power = y = 0.00114535x^2-3.4372x+2579.273
+        #for x in an8vector:
+            #do something
+
+        return 1
+
+
+    # TODO: edit powerThresh constant, import thruster_power function function, edit else return value
+    def acceptable_total_power(self):
+        # tests the total power consumed
+
+        POWER_THRESH = 1000
+
+        totalPower = sum(convertToPowerUnit(self.an8vector))
+
+        if totalPower < POWER_THRESH:
+            return 1
+        else:
+            # TODO: JASON: Need function to calculate direction scalar from power scalar.
+            return floor(POWER_THRESH/totalPower * 10) / 10
+
+    # TODO: edit ind_thresh
+    def acceptable_individual_total(self):
+        # tests whether each individual thruster is exceeding the maximum
+        # "Limiting"
+        # should return 1 if everything is acceptable, otherwise a value \in (0,1) which will be multiplied to the 6vector
```

Comment:
~\in~ => in? Just small typo I think you have here and in the other places

Reply:
fixed typos